src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionCouponsContext.php

```diff
@@ -98,7 +98,7 @@ public function specifySuffixAs(string $suffix): void
     }
 
     /**
-     * @When /^I limit generated coupons usage to (\d+) times$/
+     * @When /^I limit generated coupons usage to (\d+) times?$/
```

Comment:
```suggestion
     * @When /^I limit generated coupons usage to (\d+) time(s)$/
```
Maybe we should use this form everywhere? 🤔

Reply:
Won't work for every case since `(s)` in regex will catch a literal `s` string and try to pass it to the method, if the order of parameters is different it could blow up. Guess it could be changed into gherkin with
`@When I limit generated coupons usage to :limit time(s)`
but that's a matter of taste and not really a part of this PR.
As a complete side point, we should go through the contexts and cleanup the regexes before 2.0;
there's a lot that are not used or phrased weirdly

Reply:
Thanks for the explanation! Of course, it doesn't work in regex... what was I thinking? 🤦‍♂️